packages/js-sdk/src/session/filesystemWatcher.ts

```diff
@@ -32,18 +33,21 @@ class FilesystemWatcher {
   }
 
   // Starts watching the path that was passed to the contructor
-  async start() {
-    // Already started.
-    if (this.rpcSubscriptionID) return
+  async start(opts: CallOpts) {
+    const _start = async () => {
```

Comment:
Nitpick: Noticed inconsistent naming of "original" functions 
`start > _start` vs `open > open`
![Screen 2023-09-04 at 15 47 07](https://github.com/e2b-dev/e2b/assets/697301/b2814f8d-00e1-4bb4-8d47-8cdd035932d0)